AlphaWallet/Market/Coordinators/UniversalLinkCoordinator.swift

```diff
@@ -192,7 +202,25 @@ class UniversalLinkCoordinator: Coordinator {
             guard let recoverAddress = Address(string: ethereumAddress.address) else { return false }
             let contractAsAddress = Address(string: signedOrder.order.contractAddress)!
             if signedOrder.order.nativeCurrencyDrop {
-                self.makeTokenHolder([""], signedOrder.order.contractAddress)
+                let amt: Decimal
+                if let amount = Decimal(string: signedOrder.order.count.description) {
+                    amt = amount / 10000
```

Comment:
Is there an existing constant we can use for `10000`? If not do add something like this before the line and use the variable. It's at least a "executable" comment.

```
let someSensibleName = 10000
```